.null-ls_814785_pyproject.toml

```diff
@@ -0,0 +1,111 @@
+[tool.poetry]
```

Comment:
I don't think the addition of this file is intended :slightly_smiling_face:?

Reply:
yea wtf is going on with this!